web/packages/teleport/src/teleportContext.tsx

```diff
@@ -65,6 +66,7 @@ class TeleportContext implements types.Context {
   automaticUpgradesTargetVersion = cfg.automaticUpgradesTargetVersion;
   assistEnabled = cfg.assistEnabled;
   agentService = agentService;
+  botService = botService;
```

Comment:
Do we need to put this on the `TeleportContext`? I understand it's how it's been done previously but it means you have to `useTeleport()` in order to get access to `botService.fetchBots`, when you could just export `fetchBots` as a function and import it in the React component